shell/app/org-home/stores/org.tsx

```diff
@@ -120,6 +122,14 @@ const org = createStore({
         }
       }
     },
+    async getJoinedOrgs({ call, update }) {
+      const orgs = org.getState(s => s.orgs)
+      if (isEmpty(orgs)) {
+        return;
```

Comment:
1、condition error? !isEmpty(orgs) return; 
2、use select instead of org.getState;